resources/assets/logisticspipes/book/en_us/guides/quickstart_guide/page0

```diff
@@ -0,0 +1,2 @@
+"It's been a long time. How have you been?"
+First off, thanks for playing Logistics Pipes. A lot of changes have been made since the last release of LP in Minecraft 1.7.10. We'll try to explain some of these in this book. Read the next pages, if you'd like to know them.
```

Comment:
Please compact the quickstart guide to two or three pages

Reply:
Hahah sure